authentication/index.adoc

```diff
@@ -0,0 +1,65 @@
++
+[NOTE]
+====
+Users must specify either a default or custom OAuth client that receives and uses the token in their requests for the OAuth token.
+====
+
+* Managing cloud provider credentials using the xref:../authentication/managing_cloud_provider_credentials/about-cloud-credential-operator.adoc#about-cloud-credential-operator[Cloud Credentials Operator].
+
+[id="authorization-overview"]
+== About authorization in {product-title}
+Authorization involves determining whether a user has the required permissions to perform an action.
+
+Based on how users interact with an {product-title} cluster, there are xref:../authentication/understanding-authentication.adoc#rbac-users_understanding-authentication[three types of users]: regular users, system users, and service accounts. By default, each user type has a set of access permissions and privileges. These permissions and privileges are represented through the xref:../authentication/using-rbac.adoc#authorization-overview_using-rbac[role-based access control (RBAC) objects], such as rules, roles, and bindings.
+
+Administrators can manage authorization using the RBAC objects. You can also control access to an {product-title} cluster through xref:../authentication/using-rbac.adoc#rbac-projects-namespaces_using-rbac[projects and namespaces].
+
+An administrator can configure authorization for {product-title} through the following tasks:
+
+* xref:../authentication/using-rbac.adoc#creating-cluster-admin_using-rbac[Creating a cluster-admin user]: By default, your cluster has only one cluster administrator called `kubeadmin`. You can create another cluster administrator after defining an identity provider.
++
+[NOTE]
+====
+After creating the cluster admin user, xref:../authentication/remove-kubeadmin.adoc#removing-kubeadmin_removing-kubeadmin[delete the existing kubeadmin user] to improve cluster security.
+====
+* xref:../authentication/understanding-and-creating-service-accounts.adoc#service-accounts-managing_understanding-service-accounts[Creating service accounts]: Service accounts provide a flexible way to control API access without sharing a regular user’s credentials. A user can xref:../authentication/using-service-accounts-in-applications.adoc#service-accounts-using-credentials-externally_using-service-accounts[use a service account in applications] and also as xref:../authentication/using-service-accounts-as-oauth-client.adoc#using-service-accounts-as-oauth-client[an OAuth client].
+* xref:../authentication/tokens-scoping.adoc#tokens-scoping[Scoping tokens]: A scoped token is a token that identifies as a specific user who can perform only specific operations. You can create scoped tokens to delegate some of your permissions to another user or a service account.
```

Comment:
Any access token identifies a specific user so that part is redundant.

How do you scope a service account token?